### DISCUSSION TITLE
Embedded Hyperlink in .JPG Images?

### DISCUSSION THREAD

**Discussion by GlennIM** [ImageMagick/ImageMagick#1965] [General] [answered]
I'm preparing to post images to my Instagram social media account, but they do NOT allow any hyperlinks to be added to posts.

Is there a way to directly embed a hyperlink directly into the image so when posted to Instagram, when the user clicks the image he/she goes to that hyperlnk?

Thanks so much!

**Comment by fmw42** [comment, marked as answer]
No, you have to do that with HTML tags.
